tests/integration/targets/redshift/tasks/main.yml

```
@@ -119,6 +118,9 @@
         node_type: "{{ node_type }}"
         wait: yes
         wait_timeout: 1000
+        tags:
+          foo: bar
+          tizio: caio
```

I'd recommend having at least 1 tag using uppercase:
```suggestion
          Tizio: Caio
```

A number of modules use CamelCasing tricks which sometimes result in tag casing to get messed up.